django_fido/management/commands/download_authenticator_metadata.py

```diff
@@ -23,6 +24,16 @@ class InvalidCert(Exception):
     """Raised when certificate validation fails."""
 
 
+def urlsafe_b64decode(decodable: bytes) -> bytes:
+    """Alternative implementation to fill the necessary padding."""
+    m = len(decodable) % 4
+    if m == 2:
+        decodable += b'=='
+    elif m == 3:
+        decodable += b'='
+    return base64.urlsafe_b64decode(decodable)
```

Comment:
Shouldn't there be a padding added for `m == 1` as well?

Reply:
That is an invalid remainder and the string is not base64 encoded. https://tools.ietf.org/html/rfc4648#section-4

Reply:
Thanks for the clarification, you're right.